### PR TITLE
fix localhost lookup problem when ws enable

### PR DESCRIPTION
**Pull request by QingGo**

### CODE REVIEW

File: src/base/system.c [hunk 1]
```diff
@@ -1148,6 +1148,9 @@ int net_host_lookup(const char *hostname, NETADDR *addr, int types)
 		hints.ai_family = AF_INET;
 	else if(types == NETTYPE_IPV6)
 		hints.ai_family = AF_INET6;
+#if defined(CONF_WEBSOCKETS)
+	hints.ai_family = AF_INET;
```
Comment (def-) [inline src/base/system.c:1152]: that seems wrong. You're disabling ipv6 totally by doing this, right?

Comment (QingGo) [reply src/base/system.c:1152]: when using CClient::Connect

Comment (QingGo) [reply src/base/system.c:1152]: ah, but maybe this will affect other call of net_host_lookup

**Comment by QingGo** [comment]
now I change the implement. just make NETTYPE_WEBSOCKET_IPV4 have the higher priority

**Pull request merged**